src/Lti/Extractor/ReplaceResultSourceIdExtractor.php

```diff
@@ -39,7 +39,7 @@ public function extractSourceId(string $xmlContent): int
             throw new InvalidLtiReplaceResultBodyException();
         }
 
-        $xml->registerXPathNamespace('x', 'http://www.imsglobal.org/lis/oms1p0/pox');
+        $xml->registerXPathNamespace('x', 'http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0');
```

Comment:
Just to share info. It seems this is wrong since 2013. This was namespace used on final draft of LTI 1.1: 

https://tracker.moodle.org/browse/MDL-40531

Reply:
Since it has been also reported in the ticket linked to this PR, Im going to resolve this discussion, thanks for investigation